deployer/src/error.rs

```diff
@@ -38,8 +38,8 @@ pub enum Error {
 
 #[derive(Error, Debug)]
 pub enum TestError {
-    #[error("Tests failed: {0}")]
-    Failed(#[from] CargoTestError),
```

Comment:
CargoTestError was removed in the PR I referenced in the PR description, and CliError does not impl Display. I'll need to double-check that how I handled these changes is correct.